packages/host/src/AppDrawer.test.tsx

```diff
@@ -8,78 +9,83 @@ import {
 import { act } from 'react-dom/test-utils';
 
 describe('AppDrawer', () => {
-  it('Collapses when clicking the drawer open/close button for the first time on a large screen', () => {
-    render(
+  it('Collapses when clicking the drawer open/close button for the first time on a large screen', async () => {
+    const { getByRole, getByTestId } = render(
```

Comment:
these tests started failing - the `open the menu` text was changed to `close the menu` sometimes which makes sense. Also apparently `userEvent` is better to use 🤷